File: Java/greetings2.java
/*
 *	Problem: "Greetings!"
 */

import java.util.Scanner;

public class greetings2 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		String input = sc.nextLine();
		String result = "h";
		int numE = (input.length() - 2);
		for (int i = 0; i < numE; i++) {
			result += "ee";
		}
		result += "y";
		System.out.println(result);
	}
}

File: Ruby/greetings2.rb
=begin
 *
 *	Problem:		Greetings!
 *	Problem ID:	greetings2
 *	url:				https://open.kattis.com/problems/greetings2
 *	Solution:		Ruby
 *
=end

class Solver

	def solve
		greeting = gets.chomp
		result = "h" + (greeting[1..-2] * 2) + "y"
		puts result
	end
	
end

solver = Solver.new
solver.solve
